test/system/src/main/java/io/pravega/test/system/framework/services/MarathonBasedService.java

```diff
@@ -74,8 +74,7 @@ public boolean isRunning() {
             log.debug("App Details: {}", app);
             //app is not running until the desired instance count is equal to the number of task/docker containers
             // and the state of application is healthy.
-            if ((app.getApp().getTasksRunning().intValue() == app.getApp().getInstances().intValue())
-                    && app.getApp().getTasksRunning().intValue() == app.getApp().getTasksHealthy().intValue()) {
```

Comment:
The description says that we are removing this because it is adding delay and causing the test to fail, but isn't waiting on the task to be healthy the right thing to do? I'm missing some context here and some reasoning on why this is the right thing to do.

Reply:
Yes, reverted it back.